suzieq/engines/pandas/topology.py

```diff
@@ -95,7 +96,11 @@ class Services:
         if peerHostname and 'peerHostname' not in fields:
             addnl_fields.append('peerHostname')
 
-        if 'asn' in columns or asn:
+        if (asn or afi_safi) and area:
```

Comment:
Why not having this check in the `validate_get_input()` function of the sqobject?

Reply:
Ideally all these checks should be done in sqobj. What we'd need to provide are mutually exclusive arguments and have those checked to make the checks table-independent. But maybe its OK to start a version now in topology alone and let it grow to include others. OTOH, we also have to check a bunch of other things such as what the type of each input is and reject it if its of a different type. So, I'll create a separate issue to handle all input validation in sqobj and continue with this patch.